include/onnxruntime/core/graph/graph.h

```diff
@@ -1377,6 +1382,8 @@ class Graph {
   IOnnxRuntimeOpSchemaCollectionPtr schema_registry_;
 
   std::vector<std::unique_ptr<onnxruntime::Function>> function_container_;
+
+  std::unordered_map<std::string, const ONNX_NAMESPACE::FunctionProto*> model_local_functions_;
```

Comment:
Given that these functions have a name as well as domain, what is used as the key in the above map?

Reply:
I see the name is used. But I am concerned it may not be enough. It should be the pair (name, domain), either as a pair or concatenated into a string like "domain.name".

Reply:
updated the key for this map. I was using domain + name combination when searching for functions but missed to use domain here is creating the map.